pyvista/core/pointset.py

```diff
@@ -106,29 +106,84 @@ class PolyData(_vtk.vtkPolyData, PointSet, PolyDataFilters):
     - Using vertices and faces
     - From a file
 
+    Parameters
```

Comment:
For these parameters `iterable` might be too general. As far as I can tell we're mostly looking for "whatever `np.array` accepts, which should either be "array-like" (as per numpy lingo) or at least "sequence" to exclude things like strings and sets.

Reply:
Sequence works, and ideally we'd use `mypy' for static type checking.